packages/utils/src/lib/reports/generate-stdout-summary.ts

```diff
@@ -73,6 +73,8 @@ function reportToOverviewSection({
   plugins,
 }: ScoredReport): string {
   const table = new CliTable3({
+    // eslint-disable-next-line no-magic-numbers
+    colWidths: [67, 7, 8],
```

Comment:
This doesn't fit in 80 characters. 67+7+8=82. Plus there are 4 column separators, so the table is stretched to 86 characters (I checked the snapshot as well).

![image](https://github.com/code-pushup/cli/assets/34691111/9b6c49ce-7277-4926-884a-5f9e379a01fc)

If you want it to be exactly 80, then subtract 6:

```suggestion
    colWidths: [61, 7, 8],
```

If you want to calculate it relative to the `TERMINAL_WIDTH` constant, you can do something along these lines:

```suggestion
    colWidths: [TERMINAL_WIDTH - 7 - 8 - 4, 7, 8],
```

Reply:
Thx! Now it is connected to the const! Way better!